frontend/js/pages/SchemaConversionScreen/SchemaConversionScreen.page.js

```diff
@@ -16,81 +16,81 @@ import "../../services/Fetch.service.js";
 
 class SchemaConversionScreen extends HTMLElement {
 
-    get testing() {
-        return this.getAttribute("testing");
-    }
+  get testing() {
+    return this.getAttribute("testing");
+  }
 
-    connectedCallback() {
-        this.stateObserver = setInterval(this.observeState, 150);
-        Actions.showSpinner()
-        this.render();
-    }
+  connectedCallback() {
+    this.stateObserver = setInterval(this.observeState, 150);
+    Actions.showSpinner()
+    this.render();
+  }
+
+  disconnectedCallback() {
+    clearInterval(this.stateObserver);
+  }
 
-    disconnectedCallback() {
-        clearInterval(this.stateObserver);
+  sendDatatoReportTab(tableNameArray, currentTabContent) {
+    for (let i = 0; i < tableNameArray.length; i++) {
+      let filterdata = {
+        SpSchema: currentTabContent.SpSchema[tableNameArray[i]],
+        SrcSchema: currentTabContent.SrcSchema[currentTabContent.ToSource[tableNameArray[i]].Name],
```

Comment:
Looks like the type of TableNameArray has changed from a string to a more complex object. Was this for something SQL server specific ?

Reply:
This change is to fix an existing issue in the UI where it is unable to extract the table information when the table schema is different than the default one.